src/main/java/ratpack/zipkin/internal/RatpackCurrentTraceContext.java

```diff
@@ -0,0 +1,41 @@
+package ratpack.zipkin.internal;
+
+import brave.internal.Nullable;
+import brave.propagation.CurrentTraceContext;
+import brave.propagation.Propagation;
+import brave.propagation.TraceContext;
+import org.slf4j.MDC;
+
+public final class RatpackCurrentTraceContext extends CurrentTraceContext {
+
+  private final MDCProxy mdc;
+
+  private final TraceContext.Extractor<MDCProxy> extractor = Propagation.B3_STRING.extractor(MDCProxy::get);
+  private final TraceContext.Injector<MDCProxy> injector = Propagation.B3_STRING.injector(MDCProxy::put);
```

Comment:
I think this can all be removed.  It was basically an attempt at using the MDC context for span context propagation.  It isn't really working as intended.  Will switch to the Ratpack MutableRegistry approach instead.